src/main/resources/admin.vm

```diff
@@ -5,40 +5,44 @@
     <meta name="admin.active.section" content="admin_plugins_menu"/>
   </head>
   <body>
-  	<h1>LGTM Add-on Configuration</h1>
-  	<h3>Webhook Endpoint</h3>
-  	<p>The webhook URL for entry into issue tracker configuration in LGTM Enterprise.</p>
-  	<p id="webhookUrl"></p>
-  	<h3>Settings</h3>
-	<div id="message-context"></div>
-    <form id="admin" class="aui">
-      <div class="field-group">
-        <label for="secret">$i18n.getText("lgtm-servlet.admin.secret.label")</label>
-        <input type="text" id="secret" name="secret" class="text">
-      </div>
-      <div class="field-group">
-        <label for="externalHookUrl">$i18n.getText("lgtm-servlet.admin.externalHookUrl.label")</label>
-        <input type="text" id="externalHookUrl" name="externalHookUrl" class="text">
-      </div>
+    <h1>LGTM add-on configuration</h1>
+    <form id="admin" class="aui" style="width:2000px;">
```

Comment:
This style attribute seems strange. The CSS file seems to contradict the rather generous width anyway.